app/models/search_range.rb

```diff
@@ -1,49 +1,46 @@
-# ES UPGRADE TRANSITION #
-# Remove class
-class Search < ApplicationRecord
```

Comment:
Should we have a migration to drop the "searches" table?

Reply:
We can make a separate issue for that, I'd say